packages/odyssey-storybook/src/components/odyssey-labs/FileUpload/FileUpload.stories.tsx

```diff
@@ -0,0 +1,100 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { FileUpload, fileUploadVariants } from "@okta/odyssey-react-mui/labs";
+import { Meta, StoryObj } from "@storybook/react";
+// import { userEvent, within } from "@storybook/testing-library";
+// import { expect } from "@storybook/jest";
+
+import { MuiThemeDecorator } from "../../../../.storybook/components";
+import { fieldComponentPropsMetaData } from "../../../fieldComponentPropsMetaData";
+// import { axeRun } from "../../../axe-util";
+
+const storybookMeta: Meta<typeof FileUpload> = {
+  title: "Labs Components/FileUpload",
+  component: FileUpload,
+  argTypes: {
+    type: {
+      control: "boolean",
+      description: "If `multiple`, the user can upload multiple files",
+      table: {
+        type: {
+          summary: "boolean",
+        },
+      },
+    },
```

Comment:
The `type` prop is a boolean? Or is it a set of 2 strings?

Reply:
It's 2 strings. Need to fix the docs here. Good call